packages/core/src/nativeModulesTypes.ts

```diff
@@ -34,13 +34,13 @@ export type DdNativeSdkType = {
      * Sets the global context (set of attributes) attached with all future Logs, Spans and RUM events.
      * @param attributes: The global context attributes.
      */
-    setAttributes(attributes: object): Promise<void>;
+    setAttributes(attributes: object): Promise<Record<string, unknown>>;
```

Comment:
Should we document what these are returning?

Reply:
actually that is a change I forgot to revert